### DISCUSSION TITLE
Poll: Countries for in-text npl citations extraction/consolidation

### DISCUSSION THREAD

**Discussion by cverluise** [cverluise/PatCit#7] [Poll]
Recent evidence from [Bryan et al (2019)][Bryan-2019] based on the US patent corpus suggest that front page npl citations and in text npl citations are partly unrelated. 

In this context, we consider extending our dataset to in-text citations for non US countries. As usual, we want to be as useful as possible to the community. We will prioritize countries in leading positions in the poll below ⬇️  - conditional on data availability.

[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/JP%20(Japan))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/JP%20(Japan)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/CN%20(China))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/CN%20(China)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/US%20(United%20States))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/US%20(United%20States)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/DE%20(Germany))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/DE%20(Germany)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/EP%20(European%20Patent%20Office))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/EP%20(European%20Patent%20Office)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/KR%20(South%20Korea))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/KR%20(South%20Korea)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/WO%20(WIPO))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/WO%20(WIPO)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/GB%20(United%20Kingdom))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/GB%20(United%20Kingdom)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/CA%20(Canada))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/CA%20(Canada)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/FR%20(France))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/FR%20(France)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/AU%20(Australia))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/AU%20(Australia)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/ES%20(Spain))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/ES%20(Spain)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/TW%20(Taiwan))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/TW%20(Taiwan)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/RU%20(Russia))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/RU%20(Russia)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/SU%20(USSR-Soviet%20Union))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/SU%20(USSR-Soviet%20Union)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/AT%20(Austria))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/AT%20(Austria)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/IT%20(Italy))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/IT%20(Italy)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/BR%20(Brazil))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/BR%20(Brazil)/vote)
[![](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/BE%20(Belgium))](https://api.gh-polls.com/poll/01DRXXY9CB1XFSY6ZJPERNSME2/BE%20(Belgium)/vote)

The above list is restricted to the top 20 patent offices by number of all time patents reported in the GBQ Patents Public Data (patents-public-data:patents.publication). If you miss some important country for your research, feel free to spot it in comments, I will add it to the poll asap. 

[Bryan-2019]:http://www.kevinbryanecon.com/UsersGuidetoIntextCitations.pdf

**Comment by sumit-spru** [comment]
Hi, 

Does the PatCit database cover citation made by the patents filed in the India patent office?

Thanks,
Sumit
